ansible/roles/zero_touch_rhel_user/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+- name: Create group {{ zero_touch_rhel_user_user_group }}
+  ansible.builtin.group:
+    name: "{{ zero_touch_rhel_user_user_group }}"
+    state: present
+
+- name: Create user {{ zero_touch_rhel_user_user_name }}
+  ansible.builtin.user:
+    name: "{{ zero_touch_rhel_user_user_name }}"
+    group: "{{ zero_touch_rhel_user_user_group }}"
+    password: "{{ zero_touch_rhel_user_user_password | password_hash('sha512') }}"
+    state: present
+
+- name: Enable sudoers
+  when: zero_touch_rhel_user_enable_sudoers | default(false) | bool
+  ansible.builtin.copy:
+    content: >-
+      {{ zero_touch_rhel_user_user_name }}
+      ALL=(ALL) {{ zero_touch_rhel_user_sudo_commands | default('NOPASSWD: ALL') }}
+    dest: '/etc/sudoers.d/{{ zero_touch_rhel_user_user_name }}'
+    mode: u=rw,o=r,g=r
```

Comment:
Love the symbolic mode, but, super minor `u=rw,o=r,g=r` rather than `u=rw,g=r,o=r`?

Reply:
Agree, you have a good eye

Reply:
re-ordered